CMSgov/design-system
# [NO-TICKET] drawer unit test updates

## Summary
Updating unit test coverage for Drawer component.

### Added
Added ability to see coverage through Jest.

### Changed
Updated unit tests for `Drawer` & `DrawerToggle`. This also resulted in updating `Drawer` to remove the event listener on unmount.

I also updated these tests to use react-testing-library instead of enzyme while I'm in there.

Before:
<img width="1791" alt="Screen Shot 2022-01-25 at 9 20 23 AM" src="https://user-images.githubusercontent.com/33579665/151029981-e328639e-845a-4cc6-bb50-14882d002a32.png">

After:
<img width="1791" alt="Screen Shot 2022-01-25 at 10 38 50 AM" src="https://user-images.githubusercontent.com/33579665/151030014-2adf0efb-4241-4e99-b232-b44b0126914a.png">

## How to test
Run unit tests and ensure that all pass with `yarn test`.

If you want to run coverage, you can do `yarn test-coverage`. This will print out the stats table in your terminal. If you want the pretty HTML, you can run `yarn test-coverage` and then open the file `packages/coverage/lcov-report/index.html` in your browser & interact with it.
